src/API/Reports/Categories/DataStore.php

```diff
@@ -72,12 +72,12 @@ class DataStore extends ReportsDataStore implements DataStoreInterface {
 	 */
 	protected function assign_report_columns() {
 		$table_name           = self::get_db_table_name();
-		$this->report_columns = array(
+		$this->report_columns =  apply_filters('wc_analytics_' . $this->context . '_report_columns', array(
```

Comment:
Just a quick heads up that this filter should be prefixed with `woocommerce_analytics` instead of `wc_analytics`.